### PR TITLE
Features/daniv create style for documentation page

### PR DESCRIPTION
**Pull request by DANIV18**

### CODE REVIEW

File: src/documentation/browserdialog.cpp [hunk 1]
```diff
@@ -37,6 +37,23 @@ BrowserDialog::BrowserDialog(QWidget *parent) :
...
 {
     delete ui;
+    delete mConnectionManager;
+
+    QDir dir = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation);
+
+    bool result;
+    result = dir.cd("PairStorm");
+    result = dir.cd("temp");
+
```
Comment (vsakharevych) [inline src/documentation/browserdialog.cpp:47]: results of evaluation in line 45 are overwritten by evaluation in line 46
if result has to be true if both statements are true than use:
result &= dir.cd("...");

**Review by vsakharevych — APPROVED** [review]
Please consider comment before merging

File: src/documentation/browserdialog.cpp (hunk 1 above)
Comment (valentynfaychuk) [reply src/documentation/browserdialog.cpp:47]: image
I have already **changed & tested** this.
It works fine :)

**Review by valentynfaychuk — APPROVED** [review]
Changes are compatible with develop and can be safely merged

**Pull request merged**